index.js

```diff
@@ -1497,6 +1498,23 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# ifElse :: (a -> Boolean) -> (a -> b) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, two unary functions and a value of any type
+  //. and applies the first function to the value if the predicate evaluates
+  //. to true; otherwise applies the second function to the value.
+  //.
+  // ```javascript
+  //. > S.ifElse(S.test(/light/), S.K('dark'), S.K('light'), 'light')
+  //. 'dark'
+  //.
+  //. > S.ifElse(S.test(/light/), S.K('dark'), S.K('light'), 'dark')
+  //. 'light'
+  //. ```
```

Comment:
Yes, that would be great. I wasn't sure if we could use ES6 in the examples. It would be nice to be able to use it in the tests: no build step either if we use `mocha --compilers js:babel/register`. :wink: